tools/venv_transfer_commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# Please don't try and run this script directly!
+ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "sudo -u repo.software cvmfs_server transaction"
+ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "mkdir -p /cvmfs/software.igwn.org/pycbc/${ENV_OS}/virtualenv/pycbc-${SOURCE_TAG}"
+rsync --rsh="ssh -o StrictHostKeyChecking=no" $RSYNC_OPTIONS -qraz ${VENV_PATH}/ cvmfs.pycbc@cvmfs-software.ligo.caltech.edu:/cvmfs/software.igwn.org/pycbc/${ENV_OS}/virtualenv/pycbc-${SOURCE_TAG}/
+ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "sudo -u repo.software cvmfs_server publish"
```

Comment:
There needs to be some sort of failure protection in here so that if the `rsync` fails for any reason, the `cvmfs_server publish` command is _not_ executed, but instead `cvmfs_server abort` is executed. Maybe:

```suggestion
rsync --rsh="ssh -o StrictHostKeyChecking=no" $RSYNC_OPTIONS -qraz ${VENV_PATH}/ cvmfs.pycbc@cvmfs-software.ligo.caltech.edu:/cvmfs/software.igwn.org/pycbc/${ENV_OS}/virtualenv/pycbc-${SOURCE_TAG}/ || \
{ ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "sudo -u repo.software cvmfs_server abort -f software.igwn.org"; exit 1; }
ssh -o StrictHostKeyChecking=no cvmfs.pycbc@cvmfs-software.ligo.caltech.edu "sudo -u repo.software cvmfs_server publish software.igwn.org"
```

Reply:
There is error protection via the script that wraps this.

Reply:
Does the wrapper issue `cvmfs_server abort` if the script fails?

Reply:
I think so (it's part of this PR, so you should be able to see the change in the other code edited .. I'd probably be happier with this change if you look at that bit as well, as some of how we're doing the  SSH access is in there ... The only reason I split this into a separate script was because it made the bash look less ugly for "run this command on fail" if I put the block of code in a separate script).

Reply:
..... Although now I realise that I managed to make a typo and wrote `publish` instead of `abort` 🤦

Reply:
Right, I see that if I mentally replace `publish` with `abort` in the outer script, it looks like it should.